src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -526,8 +529,11 @@ void LiveFunctionsDataView::OnDataChanged() {
 void LiveFunctionsDataView::OnTimer() {
   if (!app_->IsCapturing()) return;
 
-  const std::vector<uint64_t> missing_scope_ids = FetchMissingScopeIds();
-  indices_.insert(std::end(indices_), std::begin(missing_scope_ids), std::end(missing_scope_ids));
+  const std::vector<ScopeId> missing_scope_ids = FetchMissingScopeIds();
+
+  for (ScopeId scope_id : missing_scope_ids) {
+    AddToIndices(scope_id);
+  }
```

Comment:
Have you verified that this behaviour change is not a problem? How many ScopeIds do we usually add here in a single OnTimer call?

Reply:
The main difference is that the new version potentially makes multiple allocations. So I'll just add .reserve.